src/utils/api.js

```diff
@@ -38,47 +52,69 @@ export const getDistricts = () => {
     accept: 'application/json',
   })
     .then(checkStatus)
-    .then(parseJSON);
+    .then(parseJSON)
+    .then(data => ({ data }))
+    .catch(error => ({ error }));
 };
 
 export const getPoliticalParties = () => {
   return fetch(`${serverBaseUrl}/v1/parties/`, {
     accept: 'application/json',
   })
     .then(checkStatus)
-    .then(parseJSON);
+    .then(parseJSON)
+    .then(data => ({ data }))
+    .catch(error => ({ error }));
```

Comment:
note that this will also catch `parseJSON` errors.

The reason for mapping `data` and `error` this way, is that we can identify in the app if the request failed or not; by checking `response.data` or `response.error`